chapter 4 edits (#312)
r-causal/causal-inference-in-R: chapters/04-dags.qmd

  ggplot(aes(year, pct)) +
  geom_line(color = "#0072B2", linewidth = .9) +
  scale_y_continuous(
    labels = scales::label_percent()
  ) +
  theme(

>     axis.title = element_blank()

Reviewer: Is your thinking here that it's clear from the axis what this is? It doesn't seem that way to me the way it can be obvious from other labels.

-   a graduate student's mood, sense of humor, and how prepared they feel for the exam could influence whether they listened to a podcast the morning of the test
-   their mood and how prepared they are also influence their exam score

> Notice we *do not* see podcast in the exam equation; this means that we are constructing this DAG under the null assumption that there is *no* causal relationship between listening to the podcast and the exam score.

Reviewer: ```suggestion
Notice we *do not* see podcast in the exam equation; this means that we assume that there is *no* causal relationship between podcast and the exam score. 
This is sometimes called a **null DAG**.
```

What if we separate these ideas and make it a little more explicit? We bring up null DAGs later, so it might be useful to emphasize

Reviewer 2: I like that
